tectonic/xetexini.c

```diff
@@ -4172,9 +4172,10 @@ tt_run_engine(char *dump_name, char *input_file_name)
      */
 
     pdf_files_init();
+    /* Initialize SyncTeX before calling start_input, which calls into SyncTeX engine! */
```

Comment:
I think this comment can be removed. It's clear that synctex is being initialized, and in this kind of code where a bunch of functions are being called in a row, the default assumption has to be that the order in which they're called is the order in which they *need* to be called.